CRVResponse/src/CrvStepsFromStepPointMCs_module.cc

```diff
@@ -59,6 +59,8 @@ namespace mu2e
 	  Comment("Starting size for crvIndex-particle vector"),4};
 	fhicl::Atom<bool> removeNeutralParticles{ Name("removeNeutralParticles"),
 	  Comment("Removes steps of neutral particles"),true};
+	fhicl::Atom<bool> useTotalEDep{ Name("useTotalEDep"),
+	  Comment("Use total energy deposition instead. Visible is a new default"),false};
```

Comment:
Beyond the patch, is it ever possible TotalEDep would be the right answer?  If not, I'd prefer this option be labeled as a 'backwards compatibility' mode, together with the postPosition kludge.